assets/styles/homepage.scss

```diff
@@ -268,6 +272,7 @@ h2 {
 // Contact section extra padding so scrollspy works
 .section-end {
   @include media ($medium-screen) {
+    padding-top: 13rem;
```

Comment:
I included some extra padding on the top of the Contact section to visually balance the contact section. It was too top heavy before. I'm okay with reverting this change, NBD.